internal/artifacts/testdata/mta_with_flat_module/mta.yaml

```diff
@@ -0,0 +1,11 @@
+ID: mta
+_schema-version: '2.1'
+version: 0.0.1
+
+modules:
+- name: mod1
```

Comment:
You should define the build result name for one of the modules so they won't both be called `data.zip` inside the build result folder

Reply:
done